pkg/daemon/manager/manager.go

```diff
@@ -75,6 +78,72 @@ func (m *manager) DeleteService(name string) error {
 	return winSvc.Delete()
 }
 
+func (m *manager) DeleteServiceAndDependents(name string) error {
+	existingSvcs, err := m.GetServices()
+	if err != nil {
+		return errors.Wrap(err, "could not determine existing Windows services")
+	}
+	// Nothing to do if it already does not exist
+	if _, present := existingSvcs[name]; !present {
+		return nil
+	}
+
+	winSvcObj, err := m.OpenService(name)
+	if err != nil {
+		return errors.Wrapf(err, "unable to open %s service handler", name)
+	}
+	defer winSvcObj.Close()
+
+	status, err := winSvcObj.Query()
+	if err != nil {
+		return errors.Wrapf(err, "unable to query %s service status", name)
+	}
+	if status.State != svc.Stopped {
```

Comment:
Is there some other state that will prevent deletion?

Reply:
Not that I could tell from my reading